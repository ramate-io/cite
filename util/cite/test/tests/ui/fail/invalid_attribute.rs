// Test that invalid citation attributes fail to compile

use cite_util::{cite, mock::MockSource};

#[cite(MockSource::same("content"), invalid_attr = "value")]  // Should fail
fn test_function() {}

fn main() {}
