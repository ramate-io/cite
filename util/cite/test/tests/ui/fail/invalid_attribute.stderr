error: Unknown citation attribute: invalid_attr
 --> tests/ui/fail/invalid_attribute.rs:5:37
  |
5 | #[cite(MockSource::same("content"), invalid_attr = "value")]  // Should fail
  |                                     ^^^^^^^^^^^^

warning: unused import: `mock::MockSource`
 --> tests/ui/fail/invalid_attribute.rs:3:23
  |
3 | use cite_util::{cite, mock::MockSource};
  |                       ^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
